bin/cli.js

```diff
@@ -81,6 +99,11 @@ function rewrite(cb){
   walkdir(dir,function(p,stat){
     if(p.substr(p.length-3) !== '.js') return
     if(~p.indexOf('.git') || ~p.indexOf('node_modules')) return
+    for (var i = 0; i < ignorePaths.length; i++){
```

Comment:
Walkdir has ignore paths built in too. Anyway thanks for the fixes! Amazing
On Nov 18, 2015 5:39 PM, "jangie" <notifications@github.com> wrote:

> In bin/cli.js
> <https://github.com/npm/scope-it/pull/3#discussion_r45271968>:
>
> > @@ -81,6 +99,11 @@ function rewrite(cb){
> >    walkdir(dir,function(p,stat){
> >      if(p.substr(p.length-3) !== '.js') return
> >      if(~p.indexOf('.git') || ~p.indexOf('node_modules')) return
> > +    for (var i = 0; i < ignorePaths.length; i++){
>
> Ah, true. I think that we could do that so long as we normalized the end
> of the inputted paths such that they matched whatever is returned as it's
> being walked.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/npm/scope-it/pull/3/files#r45271968>.
>